### PR TITLE
Typo in language file

### PR DESCRIPTION
**Pull request by deathart**

### CODE REVIEW

**Comment by deathart** [comment]
On the other hand I did not understand the utility of the line:  https://github.com/deathart/CodeIgniter4/blob/8ff5bf615b9d41f6acba6693a4bc80d8b1ab71fa/system/Language/en/Images.php#L59

**Comment by lonnieezell** [comment]
@deathart - the PageNotFound text is in there for localizing to other languages.

File: system/Language/en/Time.php [hunk 1]
```diff
@@ -1,21 +1,57 @@
...
-	'inFuture' => 'in {0}',
-	'yesterday' => 'Yesterday',
-	'tomorrow' => 'Tomorrow',
-	'now' => 'Just now',
+	'invalidMonth' 		=> 'Months must be between 0 and 12.',
+	'invalidDay' 		=> 'Days must be between 0 and 31.',
+	'invalidHours' 		=> 'Hours must be between 0 and 23.',
+	'invalidMinutes' 	=> 'Minutes must be between 0 and 59.',
+	'invalidSeconds' 	=> 'Seconds must be between 0 and 59.',
+	'years' 			=> '{0, plural, =1{# year} other{# years}}',
```
Comment (lonnieezell) [inline system/Language/en/Time.php]: These should be spaces used for alignment. As you can see here on GitHub, use tabs for alignment doesn't always work :)

File: system/Language/en/Migrations.php [hunk 2]
```diff
@@ -73,6 +73,6 @@
 	'migRemoved'        => 'Rolling back: ',
 	'migAdded'          => 'Running: ',
 
-	'version'  => 'Version',
-	'filename' => 'Filename',
+	'version'  			=> 'Version',
```
Comment (lonnieezell) [inline system/Language/en/Migrations.php]: Use spaces for alignment please.

File: system/Language/en/Database.php [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+ *
+ * @package	CodeIgniter
+ * @author	CodeIgniter Dev Team
+ * @copyright	2014-2018 British Columbia Institute of Technology (https://bcit.ca/)
+ * @license	https://opensource.org/licenses/MIT	MIT License
+ * @link	https://codeigniter.com
+ * @since	Version 3.0.0
+ * @filesource
+ */
+
```
Comment (lonnieezell) [inline system/Language/en/Database.php]: Would prefer to use smaller versions of the header for language files, similar to what's use in the [codeigniter3-translations](https://github.com/bcit-ci/codeigniter3-translations) project.

**Comment by lonnieezell** [comment]
I was too quick to comment earlier and didn't really notice this was the Images lang file you were pointing to. You're right - that can be deleted.

**Comment by lonnieezell** [comment]
That copyright is great. And yes, put it on all language files, please.

**Comment by deathart** [comment]
Here it is done:)

**Comment by lonnieezell** [comment]
Looks good, thanks!

**Comment by jim-parry** [comment]
@deathart ps - we expect contributions to be gpg-signed, thanks

**Pull request merged**